internal/pkg/cli/interfaces.go

```diff
@@ -232,7 +232,7 @@ type wsAppManager interface {
 }
 
 type wsAddonManager interface {
-	WriteAddon(f encoding.BinaryMarshaler, svc, path string) (string, error)
+	WriteAddon(f encoding.BinaryMarshaler, svc, storageName string) (string, error)
```

Comment:
I wonder if we should have the parameter be just `name` instead? In case in the future we add commands for other types of addon like a stream or queue